### PR TITLE
Feature/network saving

### PR DESCRIPTION
**Pull request by ManiacMaxo**
Pickling the parameters of a Flow using numpy

### CODE REVIEW

File: nujo/utils/flow_serialization.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from numpy import array, load, save
+
+from nujo.flow import Flow
+
+__all__ = ['save_flow', 'load_flow']
+
+
+def save_flow(flow: Flow) -> None:
+    '''Save flow
```
Comment (VIVelev) [inline nujo/utils/flow_serialization.py]: Missed a space

File: nujo/utils/flow_serialization.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    Saves the parameters of a Flow to a file named:\n
+    *name of flow*_parameters.npy in the current directory.
+
+    Parameters:
+    -----------
+    Flow : will take parameters and save them to a .npy file
```
Comment (VIVelev) [inline nujo/utils/flow_serialization.py]: Missed a `-`

File: nujo/utils/flow_serialization.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    Load the parameters of a Flow from a file
+
+    Parameters:
+    -----------
+    - Flow : this flow will have its parameters set to the read ones
+    - filename : the file path to get array from
+
+    '''
+    return Flow(_chain=load(filename, allow_pickle=True))
```
Comment (VIVelev) [inline nujo/utils/flow_serialization.py]: Have you actually tested this? I am skeptical of this working as expected.

**Review by VIVelev — CHANGES_REQUESTED** [review]

File: nujo/utils/flow_serialization.py (hunk 3 above)
Comment (ManiacMaxo) [reply nujo/utils/flow_serialization.py]: It does work, I have tested

**Review by VIVelev — APPROVED** [review]
good

**Pull request merged**